src/lib/plot/axis/axis.js

```diff
@@ -140,18 +145,20 @@ class Axis extends PureRenderComponent {
     };
 
     const {
-      left,
-      top,
-      width,
+      className,
       height,
+      left,
       orientation,
-      title
+      title,
+      top,
+      width
     } = props;
-
+    const isY = [LEFT, RIGHT].indexOf(orientation) > -1;
```

Comment:
I typically recommend `.includes` for bit of added readability, if there are existing usages of `.includes`, we should use that instead since that would mean existing users have it already set up.